tests/integration/localstorage/localstorage_int_test.go

```diff
@@ -72,6 +72,10 @@ var _ = Describe("local storage", func() {
 			fileStat, err = os.Stat(k3sStorage + "/" + volumeName)
 			Expect(err).ToNot(HaveOccurred())
 			Expect(fmt.Sprintf("%04o", fileStat.Mode().Perm())).To(Equal("0777"))
+
+			touchResult, err := testutil.K3sCmd("kubectl --namespace=default exec -it volume-test -- touch /data/file")
```

Comment:
Lets have the pod itself do this

Reply:
I'm not sure why it would be flakey, are you concerned that the test `kubectl exec` command might run before the pod has entered the sleep? If so, just wait for a second or two after the pod becomes ready before running the test.

I do want to see the pod command itself successfully writing a file to the PVC, not just the test command.